src/main/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltration.java

```diff
@@ -369,6 +369,7 @@ private void filter(final VariantContext vc, final FeatureContext featureContext
         }
 
         // make a new variant context based on filters
+        //Note that making this empty set effectively converts the VC to PASS, whereas an unfiltered VC has null filters
```

Comment:
We're in general NOT good about treating PASS and unfiltered differently.  I'm pretty sure htsjdk conflates the two as "not filtered"

Reply:
Let me test...

Reply:
Well, it seems we do track it.  nevermind.